.github/workflows/tests_extra.yml

```
@@ -59,10 +59,16 @@ jobs:
       artifact-path: |
         results.xml
       envs: |
+        - linux: check-docs-links
+        - macos: py311-xdist
+          pytest-results-summary: true
         - macos: py312-xdist
           pytest-results-summary: true
-        - linux: py311-xdist
+        - linux: py3-stdevdeps-xdist
```

Note for the next PR?
```suggestion
        # TODO: Merge stdevdeps into devdeps
        - linux: py3-stdevdeps-xdist
```